src/legacy/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -93,6 +93,20 @@ uiRoutes
           ? getSavedSearchBreadcrumbs
           : getRootBreadcrumbs
       ),
+    badge: (i18n, uiCapabilities) => {
+      if (uiCapabilities.discover.save) {
+        return undefined;
+      }
+
+      return {
+        text: i18n('kbn.discover.badge.readOnly.text', {
+          defaultMessage: 'Read only',
+        }),
+        tooltip: i18n('kbn.discover.badge.readOnly.tooltip', {
+          defaultMessage: 'Unable to save saved searches',
```

Comment:
Is the word "saved" necessary here?

Reply:
We've referred to the actual "saved object type" elsewhere as "saved searches", but I do agree that this language is super awkward. I'm not opposed to "Unable to save searches"